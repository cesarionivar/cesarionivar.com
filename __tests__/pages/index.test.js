import { render } from '@testing-library/react';
import Home from '../../pages';

describe('Home', () => {
  test('it renders HomePage unchanged', () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
