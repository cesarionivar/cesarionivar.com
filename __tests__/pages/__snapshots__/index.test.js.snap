// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home it renders HomePage unchanged 1`] = `
<div>
  <div
    class="landing page"
  >
    <div>
      <div
        class="banner"
      >
        <img
          alt="Welcome"
          class="welcome-img"
          src="/img/greet.png"
        />
        <br />
        <h2
          class="title"
        >
          &lt; I'm a Software Developer /&gt;
        </h2>
        <p
          class="description"
        >
          Passionate about technologies and especially into programming 👨🏽‍💻.
        </p>
      </div>
      <div
        class="hero"
      >
        <img
          alt="Hero"
          class="hero-img"
          src="/img/hero.svg"
        />
      </div>
      <div
        class="intro"
      >
        <p>
          Hey there!, it's a pleasure to meet you.
        </p>
        <p>
          “Simplicity is the soul of efficiency.” – Austin Freeman
        </p>
      </div>
    </div>
    <div
      class="resume container"
    >
      <div
        class="resume__works"
      >
        <h2>
          👀 Watch parts of the works that I haved made 
        </h2>
        <a
          href="/portfolio"
        >
          Go to portfolio
        </a>
      </div>
      <div
        class="resume__document"
      >
        <h2>
          Click the button below to get a copy of my resume 👇🏽
        </h2>
        <a
          href="https://github.com/cesarionivar/public-resources-online/raw/main/curriculum/CV%20-%20Cesario%20Nivar.pdf"
        >
          Get the resume
        </a>
      </div>
    </div>
    <div
      class="hireMe"
    >
      <div
        class="hireMe__logo"
      >
        <img
          alt="Hire Me"
          src="/img/hireMe.svg"
        />
      </div>
      <div
        class="hireMe__info"
      >
        <p>
          If you want to build a website for your bussiness or even for your personal use, such as a blogspot, don't wait to long... and hire me ✔
        </p>
        <a
          href="/contact"
        >
          Hire me! 📝
        </a>
      </div>
    </div>
  </div>
</div>
`;
